### PR TITLE
Refactor support for foreign namespace attributes and elements (#439).

### LINKED ISSUE #439: Foreign namespace elements not reflected in syntax.
### Section 5.3.2 

> In addition to standardized extension vocabulary, a conforming document instance may contain arbitrary namespace qualified elements that reside in any namespace other than those namespaces defined for use with this specification.

This option seems not reflected in the content model description of TTML elements.

### PR DESCRIPTION
**Pull request by skynavga**
Closes #439.

Marking as editorial since it has no impact on conformance.

### CODE REVIEW

File: spec/ttml2.xml [hunk 1]
```diff
@@ -18441,7 +18419,10 @@ however, in this case the author wishes to segregate certain metadata
 attributes by expressing them indirectly on <el>metadata</el>
 elements.</p>
 </note>
-<p></p>
+<note role="clarification"><p>The presence of foreign namespace attributes in a concrete encoding of a TTML
+<loc href="#terms-document-instance">document instance</loc> do not affect the formal validity of that
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: This is a little awkward grammatically, since the "presence" is a singular thing so I'm expecting "does not affect" rather than "do not affect". Suggest rewording to:

_Foreign namespace attributes that may be present in a concrete encoding of a TTML document instance do not affect..._

File: spec/ttml2.xml [hunk 2]
```diff
@@ -18468,7 +18449,10 @@ elements.</p>
 <p>In the above example, a number of elements defined by the Dublin
 Core metadata vocabulary are used to express document level metadata.</p>
 </note>
-<p></p>
+<note role="clarification"><p>The presence of foreign namespace elements in a concrete encoding of a TTML
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: Similar point to that above - suggest: 

_Foreign namespace elements that may be present in a concrete encoding..._

Comment (skynavga) [reply spec/ttml2.xml]: Fixed.

Comment (skynavga) [reply spec/ttml2.xml]: Fixed.

**Comment by skynavga** [comment]
@nigelmegitt addressed your comments, please approve
